### ISSUE TITLE
Wheel Parts not behaving properly

### ISSUE THREAD

**Issue by ghost** [Signatum/SXT#15]
Trying to use a basic TG-T1 and TG-T2 wheel block to drive around the KSC however when the wheels touch the runway the craft explosively disassembles. Other wheels seem to behave fine, the only issues are with the TG-T2 wheel well that I've tested. I haven't tested other wheel wells.

**Comment by PhineasFreak** [comment]
The wheels will need a complete overhaul to work with KSP 1.1, they are not currently compatible.

**Comment by Ruedii** [comment]
Yes, the wheels will need a complete overhaul, at least of their configuration files.

The new wheel module also handles long suspension throws better.  This might be something to consider in design.

Also, transitioning to a "double wheel module" design where there are separate left and right wheels on the wheel blocks of the trucks might be a good idea.

We may also want to check if someone has made an updated version of FireSpitter wheel module for the new wheel engine.  That may be an option instead. 

It seems a lot of modders want a new advanced wheel module.  Maybe we should try to pool together a colaberation team with the know-how and manpower to get it done.

I know the B9 team wants it done, but has nobody capable.  (I've been following that thread.)

**Comment by Ruedii** [comment]
OK, I've taken a look at the wheel module and it is largely out of date. 

It is completely missing the wheel suspension module and it might need updates to the torque module and the addition of a speed limiter.

I suspect we should update the wheel steering behavior as well. I think we should be able to do dual-mode steering with both torque-based and turn based steering on the trucks now.
